sdk/sdk-view/src/test/java/mil/emp3/api/ApplyTest.java

```diff
@@ -69,6 +69,14 @@ public void test6() throws EMP_Exception, InterruptedException {
         Thread.sleep(1000, 0);
         Assert.assertTrue("Feature p1 should be added to remoteMap[0]", mapInstance[0].validateAddFeatures(p1));
 
+        p1.apply(null);
+        Thread.sleep(1000, 0);
```

Comment:
is there an event that you can wait for when a feature is removed from the map?

Reply:
The delay is in there for the StorageManager to complete the apply operation.  It was there in the original test, and I kept the same delay for the newly added tests.  1000 ms is excessive for one feature, I agree.  However, for a bulk feature apply, there is a loop, which has a 100 ms delay within the loop.